Codecademy/gamut
# feat(ScoreSummary): create ScoreSummary gamut-labs component

### Overview


create ScoreSummary component in gamut-labs for use on cumulative assessment summary page and path syllabi


### PR Checklist

- [ ] Related to JIRA ticket: [EGG-2787]

#### Testing Instructions
check out the storybook page: http://localhost:6006/?path=/docs/brand-labs-organisms-scoresummary--score-summary


[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/33310)  | [Monolith Env](https://pr-33310-monolith.dev-eks.codecademy.com/) |





[EGG-2787]: https://codecademy.atlassian.net/browse/EGG-2787?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
